src/v2/Components/NavBar/Menus/DropDownMenu/DropDownSection.tsx

```diff
@@ -14,7 +14,7 @@ export const DropDownSection: React.FC<DropDownSectionProps> = ({
 
   return (
     <>
-      <Box width={[110, 135, 135, 170, 170]} py={4} mr={[1, 1, 2, 2]}>
```

Comment:
@deliciousnachos - re nav bar implementation, so strange that the drop down menu widths were so specific /shrug

Reply:
Ah thanks for showing me this! I knew something uncool was happening 😅